README.md

```
@@ -0,0 +1,1368 @@
+# Holidays
+
+A fast, efficient Python library for generating country- and subdivision- (e.g. state or province)
+specific sets of government-designated holidays on the fly. It aims to make determining whether a
+specific date is a holiday as fast and flexible as possible.
+
+<table>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: table

(MD033, no-inline-html)